loading/README.md

```diff
@@ -0,0 +1 @@
+# loading
```

Comment:
I know it's a WIP

But I leave a comment sure this README file will be populated

Reply:
I am splitting that big one into small PRs. The WIP one was a starter for me to set the overall goal.